### PR TITLE
Bump VERSION to 1.1.90

### PR DESCRIPTION
**Pull request by aanm**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4645)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: Documentation/contributing.rst [hunk 1]
```diff
@@ -1861,6 +1861,8 @@ For the final release
 #. Follow the :ref:`generic_release_process` to create the final replace and replace
    ``X.Y.0-rcX`` with ``X.Y.0``.
 
+#. Update the ``VERSION`` file to contain ``vX.Y.90``
```
Comment (tgraf) [inline Documentation/contributing.rst]: The step is documented a few lines above when we branch off the release branch

**Review by tgraf — CHANGES_REQUESTED** [review]

File: Documentation/contributing.rst (hunk 1 above)
Comment (aanm) [reply Documentation/contributing.rst]: @tgraf removed

**Comment by aanm** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Pull request merged**